compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangParserListener.java

```diff
@@ -59,7 +59,7 @@
         this.pkgBuilder = new BLangPackageBuilder(context, compUnit);
         this.diagnosticSrc = diagnosticSource;
         String distributedTxEnabledProp = System.getProperty(DISTRIBUTED_TRANSACTION_ENABLED);
-        if (distributedTxEnabledProp != null && !distributedTxEnabledProp.isEmpty()) {
+        if (distributedTxEnabledProp != null) {
```

Comment:
This and the above approach (BLangProgramRunner) of checking is dtx enabled is different. Bring the implementation to a common function and use it from all the places.

Reply:
Fixed with 052fbe446ddc7a23a4a7f62411a3875bfe64d96f